packages/manager/modules/dbaas-logs/src/logs/detail/inputs/add/edit/logs-inputs-add-edit.html

```diff
@@ -128,21 +128,78 @@
         </oui-field>
 
         <oui-field
-            label="{{ ::'inputs_logs_edit_nb_instance_field' | translate }}"
+            label="{{ ::'inputs_autoscaled_detailled' | translate }}"
             size="xl"
-            help-text="{{ ctrl.showInputPrices() }}"
+            help-text="{{ ::'inputs_autoscaled_explanation' | translate }}"
         >
-            <oui-numeric
-                id="input_nbInstance"
-                name="input_nbInstance"
-                data-model="ctrl.input.data.info.nbInstance"
-                min="ctrl.LogsConstants.NB_INSTANCE_MIN"
-                max="ctrl.LogsConstants.NB_INSTANCE_MAX"
-                required
+            <oui-switch
+                name="input_autoscale"
+                data-model="ctrl.input.data.info.autoscale"
             >
-            </oui-numeric>
+            </oui-switch>
         </oui-field>
 
+        <div class="pl-4 mb-5">
+            <oui-field
+                label="{{ ::'inputs_logs_edit_nb_instance_field' | translate }}"
+                size="xl"
+                data-ng-hide="ctrl.input.data.info.autoscale"
```

Comment:
**nitpick:** `data-ng-if` is a better option. `ng-hide` does not remove the element from the DOM.  This makes DOM heavy and can have unexpected blank spaces in the page.

But are you sure that `ng-hide` is required over `ng-if`?

Reply:
Updated: I changed all the 4 `data-ng-hide` by `data-ng-if`. Tested locally, works as expected